proof/proof.go

```diff
@@ -7,10 +7,25 @@ package proof
 
 import (
 	"errors"
+	"hash"
 
 	"github.com/dedis/crypto"
 )
 
+// NOTE This suite is the same as the global one. Writing it is not an easy when you
+// don't know what methods you want to have (i.e.  the same set of methods that
+// edwards25519 implements etc).
```

Comment:
Well, the original Suite is supposed to not exist anymore ! I was lucky to still have it somewhere but if it does not exists, I have to find the right signature somewhere.
Also, this methodology of "put everything, then remove one method, see if it works, and iterate again" to me looks more like a debugging methodology than a proper coding methodology. Every dev./user of the library would potentially have to follow this methodology.